kryo-serializer/src/test/java/org/vertexium/serializer/kryo/KryoVertexiumSerializerTest.java

```diff
@@ -218,24 +244,23 @@ public void before() {
             111, 32, 99, 111, 108, 108, 101, 99, 116, 105, 111, 110, 32, 119, 105, 116,
             104, 32, 100, 101, 115, 99, 114, 105, 112, 116, 105, 111, -18, 1, 1, 106,
             97, 118, 97, 46, 117, 116, 105, 108, 46, 65, 114, 114, 97, 121, 76, 105,
-            115, -12, 1, 2, 1, 1, 1, 4, -9, 7, 1, 0, 0, -128, 125, 63,
-            53, 94, -70, -71, 70, 64, -80, 114, 104, -111, -19, 76, 78, 64, -9, 7,
-            1, 0, 0, -128, 55, -119, 65, 96, -27, 48, 67, 64, -80, 114, 104, -111,
-            -19, -84, 81, 64, -9, 7, 1, 0, 0, -128, 119, -66, -97, 26, 47, -83,
-            65, 64, -14, -46, 77, 98, 16, 72, 76, 64, -9, 7, 1, 0, 0, -128,
-            125, 63, 53, 94, -70, -71, 70, 64, -80, 114, 104, -111, -19, 76, 78, 64,
-            1, 1, 1, 4, -9, 7, 1, 0, 0, -128, -6, 126, 106, -68, 116, 51,
-            52, 64, -80, 114, 104, -111, -19, -84, 65, 64, -9, 7, 1, 0, 0, -128,
-            -115, -105, 110, 18, -125, 64, 57, 64, -6, 126, 106, -68, 116, 51, 68, 64,
-            -9, 7, 1, 0, 0, -128, 84, -29, -91, -101, -60, 64, 57, 64, -86, -15,
-            -46, 77, 98, 64, 73, 64, -9, 7, 1, 0, 0, -128, -6, 126, 106, -68,
-            116, 51, 52, 64, -80, 114, 104, -111, -19, -84, 65, 64, 1, 1, 1, 4,
-            -9, 7, 1, 0, 0, -128, -27, -48, 34, -37, -7, 62, 40, 64, -106, 67,
-            -117, 108, -25, 59, 55, 64, -9, 7, 1, 0, 0, -128, 92, -113, -62, -11,
-            40, 44, 65, 64, 119, -66, -97, 26, 47, 29, 84, 64, -9, 7, 1, 0,
-            0, -128, 127, 106, -68, 116, -109, 72, 76, 64, 8, -84, 28, 90, 100, -21,
-            80, 64, -9, 7, 1, 0, 0, -128, -27, -48, 34, -37, -7, 62, 40, 64,
-            -106, 67, -117, 108, -25, 59, 55, 64, 69, 78, -60
+            115, -12, 1, 2, 1, 1, 1, 4, -21, 7, 1, 0, -128, -47, 34, -37,
```

Comment:
Still need changes?

Reply:
Nope. I've got it worked out properly now after a few changes.